Seems a new error (irrelative to this PR) happen:
```
E   tensorflow.python.framework.errors_impl.AlreadyExistsError: Another metric with the same name already exists.
```
This happen under TF 2.6 installed (it will install a incompatible keras version (keras 2.7) as dependency)

--- a/tests/tensorflow_autolog/test_tensorflow2_autolog.py
+++ b/tests/tensorflow_autolog/test_tensorflow2_autolog.py
@@ -914,6 +914,7 @@ def test_import_tensorflow_with_fluent_autolog_enables_tf_autologging():
     # so the original keras autologging is disabled
     if Version(tf.__version__) >= Version("2.6"):
         import keras  # pylint: disable=unused-variable,unused-import
+
         assert autologging_is_disabled(mlflow.keras.FLAVOR_NAME)
 
 

